test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -151,7 +151,12 @@ function vars_state(m::EDMF, st::GradientFlux, FT)
     @vars(
         S²::FT, # should be conditionally grabbed from atmos.turbulence
         environment::vars_state(m.environment, st, FT),
-        updraft::vars_state(m.updraft, st, FT)
+        updraft::vars_state(m.updraft, st, FT),
```

Comment:
is there a reason not to add these variables in the environment in 

https://github.com/CliMA/ClimateMachine.jl/blob/8e2ef08353183dbf6daaf3e5e12d3f5b5a1db0f9/test/Atmos/EDMF/edmf_kernels.jl#L135

Reply:
Good point, I will add them to the environment